### PR TITLE
separate cmd file, modify readme

### PR DESCRIPTION
**Pull request by paying4529-zz**
- complete version
- add cmd file
- modify readme

### CODE REVIEW

File: main_cmd.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import argparse
+import os
+import subprocess
+
+from hummingbird_cmd import HummingBird
+
+
+LOCAL_PATH = os.path.join(os.path.dirname(__file__), "output_reports")
+if not os.path.exists(LOCAL_PATH):
+  os.makedirs(LOCAL_PATH)
```
Comment (p9n) [inline main_cmd.py]: Skip this if user specified another output path.

File: hummingbird_cmd.py [hunk 2]
```diff
@@ -0,0 +1,1157 @@
+"""HummingBird I2C Eletrical Test Automation.
```
Comment (p9n) [inline hummingbird_cmd.py]: Don't duplicate code (especially this size).

A possible way is put main logic in  hummingbird_cmd.HummingBird, and then let
hummingbird_saleae.HummingBird inherits both AnalogMeasurer and hummingbird_cmd.HummingBird.

**Comment by paying4529-zz** [comment]
- only the overwrite function left
- super self in first priority
- only mkdir when args.out is None

File: main_cmd.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+from hummingbird_cmd import HummingBird
+
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser()
+  parser.add_argument("csv", help="csv data path")
+  parser.add_argument("--output_folder", default=None,
+                      help="output report folder")
+  parser.add_argument("--working_voltage", default=None, type=float,
```
Comment (p9n) [inline main_cmd.py:20]: You might want to add [choices=list-of-allowed-values](https://docs.python.org/3/library/argparse.html#choices) here and for operation_mode

**Review by p9n — COMMENTED** [review]
(github doesn't allow comment on unchanged lines...)

remove the `LOCAL_PATH = os.path.join(tempfile.gettempdir(), "output_reports"); ...`  lines in generate_report.py

**Review by p9n — APPROVED** [review]

**Pull request merged**